projects/core/src/configurator/commons/connectors/configurator-commons.adapter.ts

```diff
@@ -46,6 +46,15 @@ export abstract class ConfiguratorCommonsAdapter {
     parameters: Configurator.AddToCartParameters
   ): Observable<CartModification>;
 
+  /**
+   * Abstract method to read a configuration for a cart entry
+   *
+   * @param parameters add to cart parameters object
```

Comment:
Shouldn't this be the read cart parameter object?

Reply:
Correct